src/App.js

```diff
@@ -1,24 +1,92 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { useState } from 'react';
 import './App.css';
+import Contacts from './contacts';
+
+function ShowContacts() {
+  const [contacts, setContacts] = useState(Contacts.splice(0, 5));
+
+  const randomContactHandler = () => {
+    const contactCopy = [...contacts];
+    const getRandomContact = () =>
+      Contacts[
+        Math.floor(
+          Math.random() * (Contacts.length - (contactCopy.length - 1)) +
+            (contactCopy.length - 1)
+        )
+      ];
+    contactCopy.push(getRandomContact());
```

Comment:
Okay, this looks good.
On line 6, the Contacts are spliced that are not supposed to be in the array anymore.
Could that also be applicable here? Or is it desirable to have the same actor multiple times in the list?